apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -148,13 +126,14 @@ export async function updateThreadUsersSubscriptions(message, room, replies) {
 
 	await Subscriptions.setOpenForRoomIdAndUserIds(message.rid, repliesPlusSender);
 
-	await Subscriptions.setLastReplyForRoomIdAndUserIds(message.rid, repliesPlusSender, new Date());
+	// TODO: Fix this cause is waiting for a single string and it should (?!) be an array
```

Comment:
Can't you change the type? Looks like this method is only used here, I don't think it would be a dangerous change

Reply:
Updated replies types in some places: https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/f263593cbf65395b9934af6707c685e497db6cb9 and https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/8692b50e59a9078bd00376aa377e72e95bdd5564.